src/analytics.js

```diff
@@ -56,3 +57,28 @@ export function triggerAnalyticsEvent(nodeOrDoc, eventType, opt_vars) {
     analytics.triggerEvent(eventType, opt_vars);
   });
 }
+
+/**
+ * Helper method to create analytics element for specific root.
+ * @param {BaseElement} baseInstance
```

Comment:
This should just always be `Element`, not `BaseElement`. I.e. actual DOM element.